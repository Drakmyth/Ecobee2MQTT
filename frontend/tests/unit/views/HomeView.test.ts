import { shallowMount } from '@vue/test-utils';
import HomeView from '@/views/HomeView.vue';

describe('HomeView.vue', () => {
    test('renders correctly', () => {
        const wrapper = shallowMount(HomeView);
        expect(wrapper.html()).toMatchSnapshot();
    });
});
