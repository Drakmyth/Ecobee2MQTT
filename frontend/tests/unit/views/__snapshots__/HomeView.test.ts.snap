// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeView.vue renders correctly 1`] = `
"<main>
  <!---->
</main>"
`;
